<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PurseQt</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableTransactions">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>641</width>
      <height>271</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Income/Expense</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Date</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Value</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Source/Goods</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Comment</string>
     </property>
    </column>
   </widget>
   <widget class="QComboBox" name="comboBoxActiveCurrency">
    <property name="geometry">
     <rect>
      <x>495</x>
      <y>445</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelActiveCurrency">
    <property name="geometry">
     <rect>
      <x>395</x>
      <y>445</y>
      <width>90</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Default currency:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonConvertMngr">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>440</y>
      <width>75</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Convert all</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelBalance">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Balance:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelBalanceValue">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>290</y>
      <width>100</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelResidualThisMonth">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Residual this month:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelResidualThisMonthValue">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>320</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelResidualAvgMonthValue">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>350</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelResidualAvgMonth">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Average month residual:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>666</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="contextMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionAddIncome"/>
    <addaction name="actionAddExpense"/>
    <addaction name="separator"/>
    <addaction name="actionLoad"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="contextMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <action name="actionAddIncome">
   <property name="text">
    <string>Add income...</string>
   </property>
  </action>
  <action name="actionAddExpense">
   <property name="text">
    <string>Add expense...</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load...</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
